drivers/pwm/Kconfig.test

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2021, Commonwealth Scientific and Industrial Research
```

Comment:
if establishing a dedicate shared test source tree, then a Kconfig file like this could belong in there and only be sourced for test samples.